packages/zoe/src/contractSupport/priceAuthorityTransform.js

```diff
@@ -53,6 +60,27 @@ export const makePriceAuthorityTransform = async ({
     );
   };
 
+  const oneQuote = async ({
```

Comment:
this factoring is a little confusing to me. this function is doing two pretty distinct jobs.

consider having a function that requires `timer` and `timestamp` and then isolating the "initialQuote" logic to await as necessary in order to call that.